lib/heroku-deflater/railtie.rb

```diff
@@ -12,9 +12,17 @@ class Railtie < Rails::Railtie
     end
 
     # Set default Cache-Control headers to one week.
+    CACHE_CONTROL_HEADER = 'public, max-age=604800'.freeze
+
     # The configuration block in config/application.rb overrides this.
-    config.before_configuration do |app|
-      app.config.static_cache_control = 'public, max-age=604800'
+    if Rails.version >= '5.0.0'.freeze
+      config.before_configuration do |app|
```

Comment:
There's some code duplication here, actually only the `app.config` is different. Do you mind to amend this commit?